js/customize-snapshots.js

```diff
@@ -1113,6 +1113,7 @@
 				isSameStatus = api.state( 'changesetStatus' ).get() === originalOptions.data.customize_changeset_status;
 				if ( 'customize_save' === originalOptions.data.action && isSameStatus && options.data ) {
 					options.data = removeParam( options.data, 'customize_changeset_status' );
```

Comment:
@sayedtaqui but core will (somewhat unintuitively) create a new revision every time the `customize_changeset_status` is supplied. Wasn't there a reason for the param to be removed? Did it have something to do with passing the date?